src/nyc_trees/js/src/quiz.js

```diff
@@ -94,10 +98,11 @@ function restore() {
     var state = progress.load();
     if (state && state.form) {
         // Restore radio button selections.
-        $.each(state.form, function(fieldName, value) {
-            // Assumes that only checked values are serialized.
-            var radio = $('input[name="' + fieldName + '"][value="' + value + '"]');
-            radio.prop('checked', true);
+        $.each(state.form, function(fieldName, values) {
+            $.each(values, function(__, value) {
+                var radio = $('input[name="' + fieldName + '"][value="' + value + '"]');
```

Comment:
isn't this now sometimes not a `radio`? Consider using a more generic term like `input`.